### PR TITLE
Lauching ZEP website

### PR DESCRIPTION
**Pull request by MSanKeys963**
Hi. 🙋🏻‍♂️

This PR creates a website for ZEPs at https://zarr.dev/zeps. All the incoming ZEPs will be displayed on the web pages to facilitate easy reviewing and general reading.

Thanks! 🖖🏻

### CODE REVIEW

File: _posts/2022-05-26-welcome-to-jekyll.markdown [hunk 1]
```diff
@@ -0,0 +1,29 @@
+---
```
Comment (joshmoore) [inline _posts/2022-05-26-welcome-to-jekyll.markdown:1]: This file isn't needed, eh?

**Comment by joshmoore** [comment]
NB: I'm not overly sure about having images copied multiple places (assets/images/flowchart.png, and the logo) We might want to just hotlink. I mention because when the logo changed, I had to go looking for copies of the old one. People start using them in their presentations, and it kind of leads to a mess.

see: https://github.com/zarr-developers/community/tree/main/logos

File: _posts/2022-05-26-welcome-to-jekyll.markdown (hunk 1 above)
Comment (MSanKeys963) [reply _posts/2022-05-26-welcome-to-jekyll.markdown:1]: Yes! I'll remove it. This file is created whenever someone creates a new Jekyll project.

**Comment by MSanKeys963** [comment]
> NB: I'm not overly sure about having images copied multiple places (assets/images/flowchart.png, and the logo) We might want to just hotlink. I mention because when the logo changed, I had to go looking for copies of the old one. People start using them in their presentations, and it kind of leads to a mess.
> 
> see: https://github.com/zarr-developers/community/tree/main/logos

Makes sense. I'll link the logo from GitHub then.

**Pull request merged**